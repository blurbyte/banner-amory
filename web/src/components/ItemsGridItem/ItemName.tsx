import styled from 'styled-components';
import { Colors } from '@styles/colors';

const ItemName = styled.h3`
  color: ${Colors.white};
  font-size: 1.4rem;
  font-weight: 400;
  margin: 0.6rem 0 0 0.8rem;
  padding-right: 4rem;
  line-height: 1.2;
`;

export default ItemName;
