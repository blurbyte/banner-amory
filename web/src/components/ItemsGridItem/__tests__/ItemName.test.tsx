import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import ItemName from '../ItemName';

test('renders correctly', () => {
  const component = renderer.create(<ItemName />);
  expect(component.toJSON()).toMatchSnapshot();
});
