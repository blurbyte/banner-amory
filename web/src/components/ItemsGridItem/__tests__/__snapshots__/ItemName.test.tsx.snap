// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  color: #ffffff;
  font-size: 1.4rem;
  font-weight: 400;
  margin: 0.6rem 0 0 0.8rem;
  padding-right: 4rem;
  line-height: 1.2;
}

<h3
  className="c0"
/>
`;
